nginx-controller/controller/controller.go

```diff
@@ -798,6 +886,19 @@ items:
 		if !lbc.isNginxIngress(&ing) {
 			continue
 		}
+
+		if ing.Annotations["nginx.org/mergible-ingress-type"] == "minion" {
```

Comment:
we can safely skip minions, because they don't have any Secrets

Reply:
Done.